src/plugins/template/tests/functional/op_reference/irdft.cpp

```diff
@@ -58,10 +57,12 @@ class ReferenceIRDFTLayerTest : public testing::TestWithParam<IRDFTParams>, publ
 
         inputData = {params.m_input_value};
         refOutData = {params.m_expected_value};
+
+        abs_threshold = 1e-6f;
```

Comment:
Why is required to set it in the test?

Reply:
Default `abs_threshold` in `ov::test::utils::compare` is to "strict" for this op. In CPU plugin single layer test it's set to 1e-4.